gnomad/sample_qc/pipeline.py

```diff
@@ -27,8 +27,10 @@ def filter_rows_for_qc(
     snv_only: bool = True,
 ) -> hl.MatrixTable:
     """
-    Annotates rows with `sites_callrate`, `site_inbreeding_coeff` and `af`, then applies thresholds.
-    AF and callrate thresholds are taken from gnomAD QC; inbreeding coeff, MQ, FS and QD filters are taken from GATK best practices
+    Annotate rows with `sites_callrate`, `site_inbreeding_coeff` and `af`, then applies thresholds.
```

Comment:
For imperative mood throughout the description, maybe change "applies" to "apply".